packages/rs-drive/src/drive/verify/mod.rs

```diff
@@ -15,3 +15,5 @@ pub mod state_transition;
 
 /// Represents the root hash of the grovedb tree
 pub type RootHash = [u8; 32];
+/// Simply a collection of 32 Bytes. It's just that simple
+pub type Bytes32 = [u8; 32];
```

Comment:
We already have a Bytes32... we should not have two of them.

Reply:
where? I saw a different one but it was in rs-dpp I think; and imo it doesn't make sense to depend on each other for something like Bytes32?

Reply:
They already depend on each other

Reply:
but then there's also bytes32 in rs-platform-value? should everyone be using that one?

Reply:
2899837cca0a8214ef0ed12723cbf16e8ce85f53